platformAPI.md

```diff
@@ -294,18 +294,17 @@ The Done button dismisses the external web view regardless of the web history.
 
 - (object) - Options object.
   - `url` (string) - External URL to load into the new modal web view.
-  - `returnURL` (string) - URL that the external web view should intercept and load into the original web view that had presented the external web view. The external web view modal will be dismissed and the URL will not be loaded in the external web view. The the intercepted URL can match any part of the return URL. For example the returnURL value of `"www.walmart.com"` will intercept any URL that contains the host `www.walmart.com`.
+  - `returnURL` (string) - A subset of the URL that should be intercepted and loaded into the original web view. The external web view modal will be dismissed and the URL will not be loaded in the external web view. Ignoring the query string value, the `returnURL` will match against any subset of the intercepted URL including the scheme, domain, port, and path. For example the `returnURL` value of `"https//github.com/TheHolyGrail"` will intercept the URLs `"https//github.com/TheHolyGrail/Zoot"` and `"https//github.com/TheHolyGrail/BridgeOfDeath"` but not the URL `"https//github.com/"`.
```

Comment:
This doesn't explain what the returnURL is used for. I think either here or in the general documentation for this API call, we need to explain what a returnURL is and why they need it.  For example: "Sometimes the hybrid web app may send an external site a return URL to call when the external site has completed its task.  When the external site sends this URL back through the bridge, and that URL matches a subset of protocol, host, path, and port of the value given in the returnURL parameter, the bridge will dismiss the presented modal and then load the URL sent by the external site into the hybrid web application's web view."  This is not complete, but where it should head